File: app/src/main/java/com/rapdict/takuro/rapdict/AnswerData.java
package sample.intent;

import java.io.Serializable;

public class AnswerData implements Serializable{
    int question_id;
    String answer;
    String question;
    public int getQuestion_id() {
        return question_id;
    }
    public void answerSet(int question_id, String answer, String question){
        setAnswer(answer);
        setQuestion(question);
        setQuestion_id(question_id);
    }
    public void setQuestion_id(int question_id) {
        this.question_id = question_id;
    }

    public String getAnswer() {
        return answer;
    }

    public void setAnswer(String answer) {
        this.answer = answer;
    }

    public String getQuestion() {
        return question;
    }

    public void setQuestion(String question) {
        this.question = question;
    }

}

File: app/src/main/java/com/rapdict/takuro/rapdict/AnswerData.kt
package sample.intent

import java.io.Serializable

class AnswerData : Serializable {
    var question_id: Int ?= null;
    var answer: String ?=null;
    var question: String ?=null;

    fun answerSet(rec_question_id: Int, rec_answer: String, rec_question: String) {
        answer = rec_answer
        question = rec_question
        question_id = rec_question_id
    }

}